modules/server/src/network/resolvers/connections/index.ts

```diff
@@ -0,0 +1,70 @@
+import { NetworkAggregationConfig, NetworkAggregationConfigInput } from '@/network/types';
+import { ObjectValues } from '@/utils/types';
+import axios from 'axios';
+import urljoin from 'url-join';
+
+/**
+ * Connection status types
+ */
+const CONNECTION_STATUS = {
+	OK: 'OK',
+	ERROR: 'ERROR',
+} as const;
+
+type ConnectionStatus = ObjectValues<typeof CONNECTION_STATUS>;
+
+/**
+ * Check the status of remote connections
+ *
+ * @param url - Remote connection url
+ * @returns A connection status
+ */
+const checkRemoteConnectionStatus = async (url: string): Promise<ConnectionStatus> => {
+	/**
+	 * recommended way to health check gql server is to run the `__typename` query that every server has
+	 * very small query with no additional params to, so using GET is not a concern for the GQL server
+	 * adds recommended pre flight header to make sure Apollo doesn't block request by CSRF protection
+	 */
+	const healthCheckQuery = '?query=%7B__typename%7D';
```

Comment:
Very helpful comment. Can you change the opening line comment symbol to be `/*` insted of `/**`. As written, intellisense software will use this comment block as the description of `healthCheckQuery` lol.